doorstop/cli/main.py

```diff
@@ -311,6 +311,16 @@ def _add(subs, shared):
         type=utilities.positive_int,
         help="number of items to create",
     )
+    sub.add_argument(
```

Comment:
I think would be sensible to double-check that the count is 1 to allow using this option